<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="controlpanel" native="true">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>171</width>
      <height>512</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_zoom">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Zoom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_rotate">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>420</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotate</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_updown">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>340</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Up/Down</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_leftright">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>380</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Left/Right</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="zoom">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>460</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="updown">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>340</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="leftright">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>380</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="rotate">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>420</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_color">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_thick">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Thick</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_black">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_red">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_green">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_blue">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_thin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_medium">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_heavy">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_edit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_fill">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clear">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>260</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clip">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>260</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_curve">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Curve</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_curve_order">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CurveOrder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_deletenode">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>197</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>220</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>290</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_updown_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Transformation</string>
     </property>
    </widget>
    <widget class="QComboBox" name="curveorder">
     <property name="geometry">
      <rect>
       <x>106</x>
       <y>160</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_curve_deletenode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DeleteNode</string>
     </property>
    </widget>
   </widget>
   <widget class="PaintWidget" name="paintwidget">
    <property name="geometry">
     <rect>
      <x>188</x>
      <y>11</y>
      <width>601</width>
      <height>512</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuShape">
    <property name="title">
     <string>Paint</string>
    </property>
    <addaction name="actionLine"/>
    <addaction name="actionRect"/>
    <addaction name="actionPoly"/>
    <addaction name="actionCircle"/>
    <addaction name="actionEllip"/>
    <addaction name="actionCurve"/>
    <addaction name="actionThreeD"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionNothing_here"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionNothing_here_3"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionNothing_here_4"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionNothing_here_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuShape"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindow"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionLine"/>
   <addaction name="actionCurve"/>
   <addaction name="actionRect"/>
   <addaction name="actionPoly"/>
   <addaction name="actionCircle"/>
   <addaction name="actionEllip"/>
   <addaction name="actionThreeD"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/resource/new.png</normaloff>:/icons/resource/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/resource/open.png</normaloff>:/icons/resource/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/resource/save.png</normaloff>:/icons/resource/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/resource/line.png</normaloff>:/icons/resource/line.png</iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
  </action>
  <action name="actionRect">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/resource/rect.png</normaloff>:/icons/resource/rect.png</iconset>
   </property>
   <property name="text">
    <string>Rect</string>
   </property>
  </action>
  <action name="actionPoly">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/resource/poly.png</normaloff>:/icons/resource/poly.png</iconset>
   </property>
   <property name="text">
    <string>Poly</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/resource/circle.png</normaloff>:/icons/resource/circle.png</iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
  </action>
  <action name="actionEllip">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/resource/ellipse.png</normaloff>:/icons/resource/ellipse.png</iconset>
   </property>
   <property name="text">
    <string>Ellip</string>
   </property>
  </action>
  <action name="actionCurve">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/resource/curve.png</normaloff>:/icons/resource/curve.png</iconset>
   </property>
   <property name="text">
    <string>Curve</string>
   </property>
  </action>
  <action name="actionThreeD">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/resource/cube.png</normaloff>:/icons/resource/cube.png</iconset>
   </property>
   <property name="text">
    <string>ThreeD</string>
   </property>
   <property name="toolTip">
    <string>ThreeD</string>
   </property>
  </action>
  <action name="actionNothing_here">
   <property name="text">
    <string>Nothing here!!!</string>
   </property>
  </action>
  <action name="actionNothing_here_2">
   <property name="text">
    <string>Nothing here!!!</string>
   </property>
  </action>
  <action name="actionNothing_here_3">
   <property name="text">
    <string>Nothing here!!!</string>
   </property>
  </action>
  <action name="actionNothing_here_4">
   <property name="text">
    <string>Nothing here!!!</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PaintWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>paintwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
